src/components/utility/EmoteCard.vue

```diff
@@ -312,13 +314,16 @@ interface Indicator {
 			color: gray;
 			font-size: 0.65em;
 			margin-top: 0.1em;
+
```

Comment:
Why add whitespace here?

Reply:
its not me I swear , my eslint fucks up, lemme remove em